Gemfile

```diff
@@ -17,8 +17,11 @@ gem 'webpacker', '~> 4.0'
 # Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
 gem 'turbolinks', '~> 5'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
+gem 'bootstrap', '~> 4.5.2'
```

Comment:
i think you dont need this because you add bootstrap in package.json

Reply:
Yes, you are absolutely right. Done